scss/utilities/rounded-corners-mixin.scss

```diff
@@ -11,13 +11,28 @@
   border-image-source: url('data:image/svg+xml;utf8,<?xml version="1.0" encoding="UTF-8" ?><svg version="1.1" width="5" height="5" xmlns="http://www.w3.org/2000/svg"><path d="M2 1 h1 v1 h-1 z M1 2 h1 v1 h-1 z M3 2 h1 v1 h-1 z M2 3 h1 v1 h-1 z" fill="rgb(#{red($color)},#{green($color)},#{blue($color)})" /></svg>');
 }
 
+@mixin border-image-chrome() {
+  @media all and (-webkit-min-device-pixel-ratio: 0) and (min-resolution: .001dpcm) {
+    border-image-repeat: space;
+  }
+}
+
+@mixin border-image-ff() {
+  @-moz-document url-prefix() {
```

Comment:
Could you use `@supports (-moz-appearance: meterbar)` instead of `@-moz-document url-prefix()`.
Because `@ -moz-document url-prefix` will be deleted in the near future. 🙏 
https://bugzilla.mozilla.org/show_bug.cgi?id=1035091